src/getting-started/setup-unit.md

```diff
@@ -1,46 +1,66 @@
 ---
 id: setup-unit
-title: Setup Unit
-sidebar_label: Setup Unit
+title: Setup Personium Unit
+sidebar_label: Setup Personium Unit
 ---
 
 ## Introduction
 
-You can choose various system structure like followings.
+This section describes the construction of Personium Unit. As an introduction, we will build an All-in-one personium unit with an Ansible Playbook that allows all the roles of one server on IaaS.
 
-- Any Machine(s)
-  - all-in-one
-  - 3-tier
-- Local VM
-- Local Containers
+If you want to build with other configurations, please refer to [Server Software Operator's Guide](../server-operator/README.md).
 
-In this section, we introduced one simple way that is all-in-one on IaaS VM with Ansible Playbook.
-If you want to choose other way, please see [Server Software Operator's Guide](../server-operator/README.md).
+## Precautions
 
-## Notes
+Personium Unit is based on the assumption that HTTPS access is performed in different subdomains depending on the data subject such as `https://alice.personium.example` and `https://bob.personium.example`, so the following is the basic Will be required.
 
-Personium requires followings.
+-DNS record setting with a wildcard domain name such as `*.personium.example` for the IP address accessing the web server
+-Get and set SSL certificate corresponding to wildcard domain name such as `*.personium.example`
 
-- Wild card subdomain DNS settings so that users can access Personium url like `user1.personium.example` or `user2.personium.example`.
-- SSL Server certificate settings to access by HTTPS.
+To simplify the above, it is recommended to build using public IaaS and DNS service rather than building on the local network. This section also takes steps based on that.
```

Comment:
```suggestion
To simplify the above, it is recommended to build the Personium Unit using public IaaS and DNS service rather than building on a local network. This section introduces the steps based on public IaaS and DNS service.
```